webclient/app/src/app/commons/appTemplateDialog/AppTemplateDialog.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from "react";
+import {
+    Button,
+    Box,
+    Dialog,
+    DialogActions,
+    DialogTitle,
+    IconButton,
+    TextField,
+    Typography
+} from "@mui/material";
+import AppTemplateDialogStyles from "./AppTemplateDialogStyles";
+import { useTranslation } from "react-i18next";
+import { Close } from "@mui/icons-material";
+import AppTemplateRest from "../../services/AppTemplateRest"
+import ErrorAlert from "../alert/ErrorAlert";
+
+function AppTemplateDialog(props) {
+    const { appTemplate, open, onClose, onRefresh, isCreateDialog } = props;
+    const { t } = useTranslation();
+    const [internalAppTemplate, setInternalAppTemplate] = useState(null);
+    const [alert, setAlert] = useState({"open":false, "title": "ERROR", "message": ""});
```

Comment:
I would create just one state with the error message in there. You can control the open tag by evaluating the message part and open it, when there is an error message present or not.

Reply:
I would also have some hint that the request runs into errors if no error message is available.